Update admin and demo user creation (#135)
owncloud/docs-ocis: modules/ROOT/pages/deployment/general/general-info.adoc

== Default Users and Groups

> Default users and groups are only created when you have started oCIS for the very first time, either via the runtime or if you start the xref:deployment/extensions/idm.adoc[IDM extension] outside the runtime. The same is true for demo users and groups which need an environment variable to be set _on first startup_ to get created.

Reviewer: ```suggestion
Default users and groups are only created when you start oCIS for the very first time, either via the runtime or by starting the xref:deployment/extensions/idm.adoc[IDM extension] outside the runtime. The same is true for demo users and groups which need an environment variable to be set _on first startup_ to get created.
```

NOTE: If you have not declared demo user creation on first startup, you can for the time being only empty the xref:deployment/general/general-info.adoc#define-the-ocis-data-path[oCIS base directory] which resets the system. Then you can start from scratch and enable demo user creation.

// fixme: the current implementation of the bootstrap creates the admin user and if set the demo users and adds them to the idm/boltdb file, see the idm extension. BUT if you have started ocis without demo user creation, you cant let ocis create them POST first start... https://github.com/owncloud/ocis/issues/3593 

=== Admin User

> An admin user will be created with following credentials:

Reviewer: ```suggestion
An admin user will be created with the following credentials:
```
